docs/whats-new.asciidoc

```diff
@@ -13,194 +13,123 @@ Other versions: {security-guide-all}/8.6/whats-new.html[8.6] | {security-guide-a
+*Select up to three fields for grouping alerts*
 
-Now in beta, this feature allows you to {security-guide}/alerts-ui-manage.html#group-alerts[group alerts] by rule name, host name, user name, source IP address, or any other field. Each group displays information such as the alerts' severity and how many users, hosts, and alerts are in the group, and you can expand a group to display its alerts.
+You now select up to three fields to {security-guide}/alerts-ui-manage.html#group-alerts[group alerts] by to customize your alerts view. Each group is nested in the Alerts table by order of selection.
 
 [role="screenshot"]
-image::images/group-alerts.png[Alerts table with Group alerts by drop-down]
+image::whats-new/images/8.8/group-alerts.png[Group alerts]
 
 [float]
-== Interactive investigation guides query builder UI
+== Visualization actions and inline actions added to more places in the {security-app}
 
-The new query builder UI makes it easier to configure {security-guide}/interactive-investigation-guides.html[Interactive investigation guides] (a technical preview feature introduced in 8.6). When you use it to create filters and build an investigation query, syntax is automatically added to the investigation guide editor, rendered as a clickable action button that launches the query in Timeline.
+{security-guide}/es-ui-overview.html#visualization-actions[Visualization actions], which allow you to examine {es} queries used to retrieve data throughout the {security-app} or perform actions for the selected visualization, have been added to several places in the {security-app}. Look for the Inspect button (image:whats-new/images/8.8/inspect-icon.png[Inspect icon,16,16]) or options menu (image:whats-new/images/8.8/three-dot-menu.png[Options menu icon,17,17]) in the UI. 
 
-[role="screenshot"]
-image::images/ig-investigation-query-builder.png[Add investigation query UI,600]
 
-[float]
-== Cloud Security Posture enhancements
+{security-guide}/es-ui-overview.html#inline-actions[Inline actions] are displayed when you hover over a specific data field or value and allow you to customize your view or investigate further. They've also been added to more places throughout the {security-app}, such as:
 
-*Cloud Security Posture Management (CSPM)*
+* Explore pages (Host, Network, and User pages)
+* Entity analytics (Entity Analytics Dashboard, user risk score, and host risk score features)
+* Alerts and events table
+* Event details flyout
 
-The new <<cspm, CSPM feature>> identifies security risks in your AWS account configuration by checking your infrastructure — including storage, compute, IAM, and more — against configuration security guidelines defined by the Center for Internet Security (CIS).
+[role="screenshot"]
+image::whats-new/images/8.8/inline-actions-menu.png[Inline actions menu]
 
 [float]
-== Threat intelligence enhancements
+== Cloud Security enhancements
 
-*Block hashes and malicious files*
+*New Container Workload Protection (CWP) (beta)*
```

Comment:
```suggestion
*New Container Workload Protection (beta)*
```
Trying to avoid calling it CWP because it's ambiguous between Cloud workload and Container workload protection 😭